force-app/main/default/triggers/AssignServiceAgentOnRequest.trigger

```diff
@@ -0,0 +1,15 @@
+/**  @triggername           : AssignServiceAgentOnRequest
+  *  @author                : Raghavendra Kadam
+  *  @date                  : 15-Mar-2023
+  *  @description           : This is Trigger of serviceReqTriggerHandler class responsible for populate
+                              owner on Service request object in owner field 
+  *  @modification log      : 
+  *  1. Raghavendra Kadam   :   Apex Trigger and comments has been added dated on 15-Mar-2023
+  *  2. Raghavendra Kadam   :   Apex Trigger naming convention has corrected as per standards dated on 17-Mar-2023
+*/
+trigger AssignServiceAgentOnRequest on Service_Request__c (before insert, after insert, before update, after update) {
```

Comment:
Remove the trigger events if you are not using it.

Reply:
Removed unused trigger events